cli/src/pcluster/templates/awsbatch_builder.py

```diff
@@ -589,7 +594,12 @@ def _add_manage_docker_images_lambda(self):
                         sid="CodeBuildPolicy",
                     ),
                     get_cloud_watch_logs_policy_statement(
-                        resource=self._format_arn(service="logs", account="*", region="*", resource="*")
+                        resource=self._format_arn(
+                            service="logs",
+                            account=self._stack_account,
+                            region=self._stack_region,
+                            resource="log-group:/aws/lambda/pcluster-ManageDockerImages-*",
```

Comment:
minor: I'm wondering if when building all these strings we can reference the same variable used to determine the function name so that if that gets changes this is updated accordingly. I see function_id is set to `ManageDockerImages`. Where is the pcluster prefix being set?

Reply:
Done: https://github.com/aws/aws-parallelcluster/pull/3678/commits/3b75efef68aebc7956aba8fa6bc7fdd1843cb7ac